### ISSUE TITLE
Weird chrome bug due to interaction with react

### ISSUE THREAD

**Issue by yyjhao** [ProseMirror/prosemirror#1122]
### Issue details

In Chrome 88, if we use nodeView to render react content in an atom node, and such atom node appears at the start of a line, then selection after the atom node is offset by one. Notably this doesn't happen on other browsers, nor on earlier version of chrome. However, this also doesn't happen in a pure contentEditable div without ProseMirror (see https://jsfiddle.net/mjba085e/)

### Steps to reproduce

1. Go to https://noiseless-gem-meteorite.glitch.me/ with Chrome
2. Try selecting text in the second line after "token content" with mouse (you can notice some weirdness)
3. Try selecting "ab" in the same line with mouse or keyboard, then copy with ctrl/cmd + c
4. Paste the content anywhere (in the editor or outside)
5. Notice that "bc" is pasted instead.

Source code is available in https://glitch.com/edit/#!/noiseless-gem-meteorite?path=index.js%3A1%3A0

### ProseMirror version

Latest version.

### Affected platforms

- [x] Chrome
- [ ] Firefox
- [ ] Internet Explorer
- [ ] Other

**Comment by marijnh** [comment]
Glitch is glitching out again, and I can't load that example. Could you maybe put the code on gist.github.com or some other sandbox?

**Comment by jarandmi** [comment]
@yyhao Did you find a solution for this? I get the exact same error. It startet in Chrome 88.

**Comment by yyjhao** [comment]
@jarandmi yes when you create the initial DOM, you can prefill it with a zero-width character `"\u200B"`.

**Comment by jarandmi** [comment]
> @jarandmi yes when you create the initial DOM, you can prefill it with a zero-width character `"\u200B"`.

What do you mean by the initial DOM? The initialValue to the EditorState in ProseMirror?

**Comment by jarandmi** [comment]
Ah, you ment to the nodeView itself. I solved it it ```dom.prepend("\u200b")```.
Thank you so much.

**Comment by marijnh** [comment]
This issue has been moved to [issue #1122](https://code.haverbeke.berlin/prosemirror/prosemirror/issues/1122) on the [repository's new home](https://code.haverbeke.berlin/prosemirror/prosemirror/).
